index.html

```diff
@@ -12,7 +12,21 @@
 </head>
 
 <body>
+
     <div class="container">
+
+        <header class="top-nav">
+                <img src="/img/lambda-black.png" alt="Black Lambda Logo">
+                <a class="nav-links" href="#">Home</a>
+                <a class="nav-links" href="./about.html">About</a>
```

Comment:
This doesn't seem to be linking properly may need to try just "about.html" instead of "./about.html"